v3/integrations/logcontext-v2/zerologWriter/zerolog-writer.go

```diff
@@ -0,0 +1,123 @@
+package zerologWriter
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"github.com/newrelic/go-agent/v3/internal/logcontext/nrwriter"
```

Comment:
the fact that logcontext-v2 was made it's own module means the internal from newrelic/v3 can't be imported ..

Reply:
@Meroje what do you mean?

Reply:
Sorry, I had gone from the initial error I got from failing imports.

There is a missing import for `"github.com/newrelic/go-agent/v3/internal"`, and the capitalization is mismatched for `"github.com/newrelic/go-agent/v3/internal/logcontext/nrwriter"` which prevents it from being found.

Reply:
@Meroje the problem is actually that we created a submodule in internal :(. Looks like we will have to hotfix this.

Reply:
I'm pretty sure that most of the stuff in internal is not package correctly to be protected from external imports haha. We will have a fix for this immediately :)